ads/google/a4a/docs/RTB-Implementation.md

```diff
@@ -0,0 +1,65 @@
+# AMP Ad Implementation Guide for RTB Ad Exchanges
+---
+## Objective
+ 
+This guide is designed to provide additional information for SSPs and Ad Exchanges that want to support AMP Ads in a Real-Time Bidding (RTB) environment.  The IAB's OpenRTB 2.5 spec is [here](http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf&sa=D&ust=1496436943725000&usg=AFQjCNFlig47imweXbZx-yuvW8LmlonzFw).
+ 
```

Comment:
Consider rewriting to be more clear and concise:


AMP pages must ensure all content conforms to the AMP format.  When non-AMP content is included in an AMP page, there can be a delay as the content is verified. A major benefit for AMP Ads on AMP pages is that the ad can be rendered early by splicing the ad into the surrounding AMP page, without affecting the UX of the page and without delay.